platforms/hyperledger-fabric/configuration/roles/create/refresh_certs/create_channel_block/tasks/nested_create_cli.yaml

```diff
@@ -0,0 +1,134 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+############################################################################################
+# This task creates the orderer cli and fetch and modify the config block
+############################################################################################
+
+# Create the value file for creater org first orderer
+- name: "create valuefile for cli {{ orderer.name }}-{{ org.name }}"
+  include_role:
+    name: k8_component
+  vars:
+    component_type_name: "{{ org.name | lower }}"
+    component_type: "orderer_cli_job"    
+    component_name: "cli-{{ org.name }}-{{ orderer.name }}"
+    orderer_name: "{{ orderer.name }}"
+    git_url: "{{ org.gitops.git_url }}"
+    git_branch: "{{ org.gitops.branch }}"
+    charts_dir: "{{ org.gitops.chart_source }}"
+    vault: "{{ org.vault }}"
+    fabrictools_image: "hyperledger/fabric-tools:{{ network.version }}"
+    alpine_image: "{{ docker_url }}/alpine-utils:1.0"
+    storage_class: "{{ org.name }}sc"
+    release_dir: "{{ build_path }}"
+    orderer_component: "{{ orderer.name | lower }}.{{ component_ns }}"
+    orderer_address: "{{ orderer.name | lower }}.{{ org.external_url_suffix }}:8443"
+
+# Start the cli using the value file created in the previous step
+- name: "start cli"
+  shell: |
```

Comment:
If we are starting a new cli for certain tasks, make sure to remove them after the task is completed

Reply:
@mgCepeda you can click on `Resolve conversation` once you have fixed the review comments.